guardrails/validators/pydantic_field_validator.py

```diff
@@ -35,6 +36,14 @@ def __init__(
         on_fail: Optional[Callable[..., Any]] = None,
         **kwargs,
     ):
+        warn(
+            """
+            PydanticFieldValidator is deprecated (v0.3.3); will be removed (v0.4.0).
+            Instead, use the `add_validator` function with a Guardrails validator class.
```

Comment:
Let's update the second line to something like:

> Instead, use the `add_validator` function, or create a custom Guardrails validator as shown here: https://www.guardrailsai.com/docs/concepts/validators#custom-validators

Mostly just want to highlight that those are two different valid flows/possibilities.

Reply:
Only asking users to migrate to custom validators, as discussed